packages/block-library/src/navigation/index.php

```diff
@@ -549,13 +549,13 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 					</div>
 				</div>
 			</div>',
-		$modal_unique_id,
+		esc_attr( $modal_unique_id ),
 		$inner_blocks_html,
 		__( 'Open menu' ), // Open button label.
 		__( 'Close menu' ), // Close button label.
-		implode( ' ', $responsive_container_classes ),
-		implode( ' ', $open_button_classes ),
-		$colors['overlay_inline_styles'],
+		esc_attr( implode( ' ', $responsive_container_classes ) ),
+		esc_attr( implode( ' ', $open_button_classes ) ),
+		esc_attr( $colors['overlay_inline_styles'] ),
```

Comment:
IIRC you used `safecss_filter_attr` for inline styles in other places

Reply:
_I_ didn't but I did see @jorgefilipecosta had done this in the Search block. Does that function escape or just make sure it's valid CSS styles? Or is that equivalence!?

Reply:
That function removes style rules from a style string e.g: "border: 1px solid red; margin: 0px" that are considered unsafe while and when we are passing something to a style attribute we should use that function for escaping.

Reply:
I've now updated this to use `safecss_filter_attr` for `$colors['overlay_inline_styles']`